hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/FileIOEngine.java

```diff
@@ -183,13 +178,8 @@ public void write(ByteBuffer srcBuffer, long offset) throws IOException {
   @Override
   public void sync() throws IOException {
     for (int i = 0; i < fileChannels.length; i++) {
-      try {
-        if (fileChannels[i] != null) {
-          fileChannels[i].force(true);
-        }
-      } catch (IOException ie) {
-        LOG.warn("Failed syncing data to " + this.filePaths[i]);
-        throw ie;
+      if (fileChannels[i] != null) {
+        fileChannels[i].force(true);
```

Comment:
where `sync` gets called we log this IOE if it happens? that message includes the specific path?